dashboard.el

```diff
@@ -48,6 +48,9 @@
     map)
   "Keymap for dashboard mode.")
 
+(defvar dashboard-after-initialize-hook nil
+  "Hook that is run after dashboard buffer is initialized.")
```

Comment:
Can you change this to `defcustom`? And add `:group` and `:type` as parameters! Thanks!

Reply:
Done :ok_hand: